### DISCUSSION TITLE
Configurable keyboard shortcuts

### DISCUSSION THREAD

**Discussion by adrian-cocoara-lgp** [storybookjs/storybook#28627] [Help]
### Summary

In Storybook 8 the keyboard shortcuts have changed (Go full screen, Show sidebar, Show toolbar etc.) to Alt + F, Alt + S, Alt + T.
We want to restore it to the values of Storybook 7('F', 'S', 'T'). Is there any way to configure Storybook shortcuts?
I only see there is an option to disable shortcuts in manager.js
`import {addons} from '@storybook/manager-api';

addons.setConfig({
  enableShortcuts: false
});`

**Comment by adrian-cocoara-lgp** [comment]
up

**Comment by daniel-stoian-lgp** [comment]
I have the same issue
